src/index.html

```diff
@@ -310,6 +310,7 @@ <h2 id="faq">FAQ</h2>
           <details class="FAQItem" id="what-is-rpki">
             <summary><span>What is RPKI?</span></summary>
             <p>With 800k+ routes on the Internet, it is impossible to check them manually. <a href="https://blog.cloudflare.com/rpki/">Resource Public Key Infrastructure (RPKI)</a> is a security framework method that associates a route with an autonomous system. It uses cryptography in order to validate the information before being passed onto the routers. You can <a href="https://blog.cloudflare.com/rpki/">read more about RPKI</a> on the Cloudflare blog.</p>
+            <p>On May 14th, Job Snijders from NTT will present a free <a href="https://www.brighttalk.com/webcast/5648/396013/rpki-101-with-job-snijders">RPKI 101</a> webinar.</p>
```

Comment:
will you submit another PR after May 14?

Reply:
yes